package.json

```diff
@@ -78,6 +78,9 @@
     "snapshotSerializers": [
       "pretty-format/build/plugins/ConvertAnsi.js"
     ],
+    "moduleNameMapper": {
+      "src/(.*)": "<rootDir>/src/$1"
+    },
```

Comment:
why do we want this? seems likely to only cause confusion for most tools

Reply:
So that we can import from `src` in the test files as `src/` and `test/` are at the same level in the directory instead of having to add `../..` to all imports.

Reply:
right, but is avoiding `../` worth breaking "go to definition" etc?

I don't really have a strong opinion here (nor ownership 😀) but in the same way removing the `import.all` macro makes things not require specific tooling I don't think configuring the resolver of jest and eslint (and later typescript I assume) keeps the code base as approachable as possible.

(if we want something like this, we should use https://nodejs.org/api/packages.html#subpath-imports or something that has an actual spec behind it instead of ad hoc config. That said, Jest does not support it (yet)).

Again, this is your project, not mine, so feel free to structure it how you want to 🙂

Reply:
> Again, this is your project, not mine, so feel free to structure it how you want to 🙂 Just putting my opinion out there, because what else is the internet for?

Love this 😆

---

Tbh I setup all of my Typescript projects with the ability to import from `src` to make it easy to do absolute imports and move stuff around. With vscode at least (when using TS) importing from `src` doesn't break `go to definition`, do you know if it does in other editors? 

I kind of see this PR as a precursor to switching to TS so hopefully the tooling will be better then. 

This all probably stems from my Java days and when I work in Scala and being able to treat directories as packages/namespaces regardless of source set 😝

Reply:
I've been writing Rust lately, so I feel this 😀